site/webpack-client.config.js

```diff
@@ -74,6 +82,9 @@ module.exports = {
       'fixed-data-table-2/css': path.join(__dirname, '../src/css'),
       'fixed-data-table-2': path.join(__dirname, '../src/index'),
     },
+    fallback: {
```

Comment:
The core node `url` package is no longer included by webpack by default.
But it doesn't look like we need them, so I'm setting `resolve.fallback.url` to false.